packages/instantsearch.js/src/components/SearchBox/SearchBox.tsx

```diff
@@ -204,7 +205,7 @@ class SearchBox extends Component<
             rootProps={{
               className: cssClasses.reset,
               type: 'reset',
-              title: 'Clear the search query.',
+              title: 'Clear the search query',
```

Comment:
If we're really strict about it, this is breaking. A CSS rule targeting the element by `title` would no longer work.